codespell_lib/data/dictionary.txt

```diff
@@ -50328,22 +50367,42 @@ superseed->supersede
 superseedd->superseded
 superseede->supersede
 superseeded->superseded
+superseedes->supersedes
 superseeding->superseding
+superseeds->supersedes
 suphisticated->sophisticated
 suplant->supplant
 suplanted->supplanted
 suplanting->supplanting
 suplants->supplants
+suplement->supplement
+suplemental->supplemental
 suplementary->supplementary
+suplemented->supplemented
+suplementing->supplementing
+suplements->supplements
 suplied->supplied
+suplies->supplies
 suplimented->supplemented
+supllied->supplied
 supllies->supplies
+suplly->supply
```

Comment:
Perhaps `supple` and `sully` (other verbal forms of `sully` too)?

Reply:
I added `suplle->supple` and its suffixes. I added `suplly->sully` and other verb forms.